src/main/java/step5/domain/car/Distance.java

```diff
@@ -0,0 +1,55 @@
+package step5.domain.car;
+
+public class Distance {
+    private final int distance;
+
+    public Distance(int distance) {
+        this.distance = distance;
+    }
+
+    public static Distance one() {
+        return new Distance(1);
+    }
+
+    public static Distance of(int distance) {
+        return new Distance(distance);
+    }
+
+    public Distance addOne() {
+        return Distance.of(this.distance + 1);
+    }
+
+    public Distance add(int distance) {
+        return Distance.of(this.distance + distance);
+    }
+    public String statusToString() {
```

Comment:
```suggestion
    }
    
    public String statusToString() {
```

공백또한 코드켄벤션이라고 생각합니다.